cunumeric/module.py

```diff
@@ -575,7 +575,7 @@ def asarray(a: Any, dtype: Optional[np.dtype[Any]] = None) -> ndarray:
     """
     if not isinstance(a, ndarray):
         thunk = runtime.get_numpy_thunk(a, share=True, dtype=dtype)
-        array = ndarray(shape=None, thunk=thunk)
+        array = ndarray(shape=None, thunk=thunk, writeable=a.flags.writeable)
```

Comment:
this change is causing this test failure:
```
           if not isinstance(a, ndarray):
               thunk = runtime.get_numpy_thunk(a, share=True, dtype=dtype)
   >           array = ndarray(shape=None, thunk=thunk, writeable=a.flags.writeable)
   E           AttributeError: 'list' object has no attribute 'flags'
```

Reply:
@manopapad we can do something like:
```
writeable = a.flags.writeable if isinstance(a, np.ndarray) else True
```
is `True` the correct default value for inputs converted from lists, etc. ?

Reply:
Yes, silly mistake on my part 38a0893398b88369c58653a86d0570045c4fe067